### PR TITLE
Fix: pass Safe Apps origin as url, not clientUrl

### PR DESCRIPTION
**Pull request by katspaugh**
Tx-service sends an origin for each Safe App transaction, which contains the URL of the Safe App that created a tx. This URL needs to be passed to the config-service in the URL field, not clientUrl.

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -9,6 +9,9 @@ async function bootstrap() {
     app.get<IConfigurationService>(IConfigurationService);
   const applicationPort: string =
     configurationService.getOrThrow('applicationPort');
+
+  app.enableCors();
```
Comment (iamacook) [inline src/main.ts:13]: Why is this change necessary? I'd suggest we move it to a separate PR as it is unrelated.

File: src/routes/transactions/mappers/common/safe-app-info.mapper.ts [hunk 2]
```diff
@@ -29,7 +29,7 @@ export class SafeAppInfoMapper {
 
     const [safeApp] = await this.safeAppsRepository.getSafeApps({
       chainId,
-      clientUrl: originUrl,
+      url: originUrl,
```
Comment (iamacook) [inline src/routes/transactions/mappers/common/safe-app-info.mapper.ts:32]: Let's add test coverage for this.

**Comment by katspaugh** [comment]
I'm a little confused because it logs
```
{"level":"info","message":"No Safe Apps matching the origin url https://apps-portal.safe.global/safe-claiming-app (safeTxHash: 0xdf0e307ef9013523629e9f3922b70753e1e247834d628c857ddfb18ea92b3c5b)","request_id":"c95a6cff-3cbf-46c5-9e26-66b025e5b518","timestamp":"2023-08-08T07:23:15.965Z"}
```
but still displays the correct Safe App icon and name for the transaction.

File: src/main.ts (hunk 1 above)
Comment (katspaugh) [inline src/main.ts:13]: This is unrelated but it's needed when testing the service in combination with a locally running frontend.

Comment (iamacook) [reply src/main.ts:13]: I'd suggest we make this change in a separate PR.

File: src/routes/transactions/mappers/common/safe-app-info.mapper.ts (hunk 2 above)
Comment (katspaugh) [reply src/routes/transactions/mappers/common/safe-app-info.mapper.ts:32]: There's already a test for this, but it's bogus as it was returning false positives. I don't have time right now to fix it. Feel free to make a separate PR. This fix needs to go live ASAP.

Comment (iamacook) [reply src/routes/transactions/mappers/common/safe-app-info.mapper.ts:32]: Understood. I'll create a separate PR later on.

**Review by iamacook — APPROVED** [review]
